bundler/lib/bundler/compact_index_client/updater.rb

```diff
@@ -1,46 +1,60 @@
 # frozen_string_literal: true
 
 require_relative "../vendored_fileutils"
+require "rubygems/package/digest_io"
```

Comment:
I'm not sure it is currently possible to rely on this being present since combination of Bundler and RubyGems could be almost anything supported.

Reply:
DigestIO has been around for at least 11 years. It's supported, though requiring it this way does not work. We have to `require "rubygems/package"`.

Reply:
Since we don't normally have to require Gem::Package, I wonder if it would be better to make our own DigestIO here. It's an extremely simple class.